AK/UFixedBigInt.h

```diff
@@ -0,0 +1,771 @@
+/*
+ * Copyright (c) 2021, Leon Albrecht <leon2002.la@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Checked.h>
+#include <AK/Concepts.h>
+#include <AK/Format.h>
+#include <AK/NumericLimits.h>
+#include <AK/StdLibExtraDetails.h>
+#include <AK/StdLibExtras.h>
+#include <AK/String.h>
+#include <AK/StringBuilder.h>
+
+namespace AK {
+
+template<typename T>
+requires(sizeof(T) >= sizeof(u64) && IsUnsigned<T>) class UFixedBigInt;
+
+// FIXME: This breaks formatting
+// template<typename T>
+// constexpr bool Detail::IsIntegral<UFixedBigInt<T>> = true;
```

Comment:
try this:
```suggestion
namespace Detail {
template<typename T>
inline constexpr bool IsIntegral<UFixedBigInt<T>> = true;
}
```
Not sure what treating it as an integral does for us though.

Reply:
This causes a compilation error, due to one function checking integral types by size and this being an Integral would cause that code path to be compiled, which is bad, due to that code path effectively branching for each possible integer size.

I would need to look up the exact function again

Reply:
So AK::TypeErasedParameter::get_type_from_size in AK/Format.h would be tripping then